reference/3.0/Microsoft.PowerShell.Management/Start-Service.md

```diff
@@ -1,106 +1,121 @@
+`Format-List` cmdlet to format the object as a list of its properties. The append redirection
+operator (\>\>) redirects the output to the services.txt file. The output is added to the end of the
+existing file.
 
-The second command uses the Start-Service cmdlet to start the WMI service.
-It identifies the service by using the InputObject parameter to pass the $s variable containing the WMI service object to Start-Service.
-Then, it uses the PassThru parameter to create an object that represents the starting of the service.
-Without this parameter, Start-Service does not create any output.
+### Example 4: Start a disabled service
+
+This example shows how to start a service when the start type of the service is **Disabled**.
 
-The pipeline operator (|) passes the object that Start-Service creates to the Format-List cmdlet, which formats the object as a list of its properties.
-The append redirection operator (\>\>) redirects the output to the services.txt file, where it is added to the end of the existing file.
-### Example 4
 ```
-PS C:\> start-service tlntsvr
-Start-Service : Service 'Telnet (TlntSvr)' cannot be started due to the    following error: Cannot start service TlntSvr on computer '.'.
+PS> Start-Service tlntsvr
+Start-Service : Service 'Telnet (TlntSvr)' cannot be started due to the following error: Cannot start service TlntSvr on computer '.'.
 At line:1 char:14
-+ start-service  <<<< tlntsvr PS C:\> get-wmiobject win32_service | where-object {$_.Name -eq "tlntsvr"}
++ Start-Service  <<<< tlntsvr
 
+PS> Get-CimInstance win32_service | Where-Object Name -eq "tlntsvr"
 ExitCode  : 0
 Name      : TlntSvr
 ProcessId : 0
 StartMode : Disabled
 State     : Stopped
-Status    : OK PS C:\> set-service tlntsvr -startuptype manual PS C:\> start-service tlntsvr
-```
-
-This series of commands shows how to start a service when the start type of the service is "Disabled".
-The first command, which uses the Start-Service cmdlet to start the Telnet service (tlntsvr), fails.
+Status    : OK
 
-The second command uses the Get-WmiObject cmdlet to get the Tlntsvr service.
-This command retrieves an object with the start type property in the StartMode field.
-The resulting display reveals that the start type of the Tlntsvr service is "Disabled".
-
-The next command uses the Set-Service cmdlet to change the start type of the Tlntsvr service to "Manual".
+PS> Set-Service tlntsvr -StartupType manual
+PS> Start-Service tlntsvr
+```
 
-Now, we can resubmit the Start-Service command.
-This time, the command succeeds.
+The first attempt to start the Telnet service (tlntsvr) fails. The `Get-CimInstance` command shows
+that the **StartMode** property of the Tlntsvr service is **Disabled**. The `Set-Service` cmdlet
+changes the start type to **Manual**. Now, we can resubmit the `Start-Service` command. This time,
+the command succeeds. To verify that the command succeeded, run `Get-Service`.
```

Comment:
Should the command include the tlntsvr specifically to check on? Otherwise, they'll get all services back.

Reply:
You would have to do a WMI query. This is a simpler example.

Reply:
I meant like `Get-Service tlntsvr`

Reply:
Or change the run to use, since that’s more generic.